practices/run-pair-programming-sessions.md

```diff
@@ -1,103 +1,133 @@
+While pair programming can be done remotely, this approach presents some challenges such as communication barriers, technical issues, and time zone differences.
+Teams must use suitable collaboration tools (for example, Visual Studio Live Share) and establish clear communication protocols.
+
+<ed> Edit OK? This appears to be the way MS refers to the tool. Also, I notice this is the first specific tool that is referenced. Practice files discussed automation tools and visual analytics tools, generally, but did not name any names. I realize this is tricky, as you don't want to appear to be endorsing any one specific product, but specific recommendations might be helpful for some readers. This is something to consider adding in your revisions. </ed>
 
 ## How to Improve
 
-### [Lead Workshops](/practices/lead-workshops.md)
+<ed> Suggest editing the above section header to something more specific, like "How to Implement This Practice" or "Tips on Implementing This Practice" </ed>
 
-#### Trial Period
+Help your team implement this practice by hosting a viewing party of specially curated talks, starting a book club to discuss impactful titles on this topic, facilitating a roundtable discussion about relevant topics, leading a workshop, or running a retrospective.
 
-To experiment with pair programming and evaluate its effects, start by pairing team members for a designated period, such as one or two weeks. Assign tasks to pairs, ensuring a mix of experience levels if possible. Rotate pairs frequently to encourage diverse collaboration. During each session, have one person in the "driver" role, actively writing code, while the other acts as the "navigator," providing real-time feedback and suggestions. Encourage pairs to switch roles regularly. Monitor the outcomes by gathering feedback on code quality, productivity, and team satisfaction. Additionally, observe any improvements in knowledge sharing, problem-solving efficiency, and team cohesion.
+<ed> Suggested text to introduce this section and preview what it will cover. Such text would appear in all Practices. With all Practice files, I would suggest consistently presenting the topics in this order:
+- Host viewing party
+- Start book club
+- Facilitate roundtable
+[other tips... I notice that, beyond these top three, they begin to vary] My edits below reflect this suggestion. </ed>
 
-### [Host A Retrospective](/practices/host-a-retrospective.md)
 
-#### Feedback Sessions
+### [Host a Viewing Party](/practices/host-a-viewing-party.md)
+
+#### [Async Code Reviews Are Chocking Your Company’s Throughput](https://www.youtube.com/watch?v=ZlLZEQQBcFg)
+
+In this video, principal engineer at HelloFresh Dragan Stepanovic delves into the common practice of pull/merge requests (PRs) in software development teams and their potential impact on throughput. <ed> Added detail about who Dragan is. OK? </ed>
```

Comment:
The added detail about the speaker is ok, but I worry we have too much content for this. It reduces its "skim-ability" if that makes sense?

Reply:
What if we consolidate to just: "Engineer Dragan Stepanovic delves into the common practice of pull/merge requests..." That way, we don't tether him to HelloFresh (who knows if he's even still there) and we cut down on wordiness. We know it's a video since it's under the umbrella of viewing party.

Reply:
Edited.